Xamarin.Forms.Core/NavigationPage.cs

```diff
@@ -364,6 +381,13 @@ void InsertPageBefore(Page page, Page before)
 				Pushed(this, args);
 		}
 
+		Task ResolveActiveNavigation(Task task)
+		{
+			if (task.Status == TaskStatus.RanToCompletion)
```

Comment:
It can probably be removed outright because the value should be decremented regardless, yes.